jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -9,14 +9,33 @@
 from jdaviz.core.events import AddDataMessage, SliceToolStateMessage, SliceSelectSliceMessage
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin
+from jdaviz.core.user_api import PluginUserApi
 
 __all__ = ['Slice']
 
 
 @tray_registry('cubeviz-slice', label="Slice")
 class Slice(PluginTemplateMixin):
+    """
+    See the :ref:`Slice Plugin Documentation <slice>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``slice``
+      Current slice number.
+    * ``wavelength``
+      Wavelength of the current slice.  If setting, will update automatically to the nearest
+      slice.
+    * ``setting_show_indicator``
+      Whether to show indicator in spectral viewer even when slice tool is inactive.
```

Comment:
Looks like this **only** changes the status of the indicator when it is inactive, is that the intended functionality?

Reply:
yes.  If it is set to True, the line changes from blue to orange based on whether the slice tool is active to click and drag.  If it is set to False, the line disappears when inactive (but still appears and is orange when active).  Any suggestions to clarify the wording (both here and in the UI)?

Reply:
I think just removing the word "even" should be enough. Just wanted to clarify!

Reply:
agreed that is simpler, made the change!